static/config/linter_exclude_list.txt

```diff
@@ -22,3 +22,11 @@ comment on exported function AddPeer should be of the form "AddPeer ..." (golint
 comment on exported function NotifyNode should be of the form "NotifyNode ..." (golint)
 comment on exported function TriggerTestSignal should be of the form "TriggerTestSignal ..." (golint)
 comment on exported function VerifyAccountPassword should be of the form "VerifyAccountPassword ..." (golint)
+comment on exported function ValidateNodeConfig should be of the form "ValidateNodeConfig ..." (golint)
+comment on exported function CallRPC should be of the form "CallRPC ..." (golint)
+comment on exported function StartCPUProfile should be of the form "StartCPUProfile ..." (golint)
+comment on exported function StopCPUProfiling should be of the form "StopCPUProfiling ..." (golint)
+comment on exported function WriteHeapProfile should be of the form "WriteHeapProfile ..." (golint)
+comment on exported function Notify should be of the form "Notify ..." (golint)
+type name will be used as node.NodeManager by other packages, and that stutters; consider calling this Manager (golint)
```

Comment:
I think it's more reasonable to refactor this. Can you delete this line and add a note to #200?

Reply:
I fully support this. I'll add note to #200.

Reply:
done.